test/data/test_gefs.py

```diff
@@ -0,0 +1,171 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+import shutil
+from datetime import datetime, timedelta
+
+import numpy as np
+import pytest
+
+from earth2studio.data import GEFS_FX
+
+
+@pytest.mark.slow
+@pytest.mark.xfail
+@pytest.mark.timeout(30)
+@pytest.mark.parametrize(
+    "time",
+    [
+        [datetime(year=2022, month=12, day=25)],
+    ],
+)
+@pytest.mark.parametrize(
+    "lead_time",
+    [
+        timedelta(hours=0),
+        [
+            timedelta(hours=258),
+        ],
+    ],
+)
+@pytest.mark.parametrize("variable", ["t2m", ["msl"]])
```

Comment:
Do we need to test more than one variable?

Reply:
Yep, added